migrations/versions/954f066e0e4b_moderator_report_model.py

```diff
@@ -0,0 +1,37 @@
+"""Moderator report model
+
+Revision ID: 954f066e0e4b
+Revises: 047ebdac558b
+Create Date: 2020-05-22 00:33:39.113290
+
+"""
+
+from alembic import op
+from sqlalchemy_utils import UUIDType
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '954f066e0e4b'
+down_revision = '047ebdac558b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'comment_moderator_report',
+        sa.Column('reported_by_id', sa.Integer(), nullable=False),
+        sa.Column('comment_id', sa.Integer(), nullable=False),
```

Comment:
Reorder as above. By convention user column should be `user_id`. We use other names only for secondary user column onwards.